### ISSUE TITLE
end_token_index = torch.where(labels == tokenizer.eos_token_id)[1]

### ISSUE THREAD

**Issue by flzt1212** [jsksxs360/How-to-use-Transformers#6]
end_token_index = torch.where(labels == tokenizer.eos_token_id)[1] 这一行代码运行时报错呀
